components/dropdown/dropdown-content-styles.js

```diff
@@ -42,6 +44,10 @@ export const dropdownContentStyles = css`
 		top: auto;
 	}
 
+	:host([close-action]) {
+		animation: var(--d2l-dropdown-close-animation-name) 300ms ease-out;
+	}
+
```

Comment:
We would also need to handle the `prefers-reduced-motion` case (see [below in the file](https://github.com/BrightspaceUI/core/blob/main/components/dropdown/dropdown-content-styles.js#L182) and [here](https://desire2learn.atlassian.net/wiki/spaces/Accessibility/pages/1435995795/Motion) for more details) for users that have that setting enabled

Reply:
Do you know if we see this being used in other close cases at any point? If it's going to stay local to `dropdown-menu` then it might make sense to just add a property and applicable styles to `dropdown-menu`